db/db_utils.py

```diff
@@ -21,7 +21,10 @@ def __init__(self, url, metadata=None):
             self.metadata = metadata
 
 
-class DbUtils(object):
+class DbInstance(object):
+
+    db = None
+
     def _establish_db_connection(self):
         # Establish connection
         connectionString = 'postgres://' + base64.b64decode(self._username) + ':' + base64.b64decode(self._password) + '@' + base64.b64decode(self._hostName) + '/' + self._dbName
```

Comment:
I think this connection string should be stored as a environment variable / some static variable when running the install/build script on db. We should not be creating the connection string each time here.